scripts/install-cert-macos.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
This file could maybe also be moved to `./scripts/helper`

Reply:
done